### ISSUE TITLE
vue2 use provide / inject ,get undefined

### ISSUE THREAD

**Issue by jeremyjone** [vuejs/composition-api#964]
I created a component by vue2(v2.6.14). It use `provide` and `inject` in `@vue/composition-api` to create a local state management. 

```js
export const initStore = () => {
    const data = reactive([]);
    provide('data1', data);
    ...
}
```

```js
export const useStore = () => {
    return {
        data1: inject('data1');
        ....
    }
}
```

In root component `initStore`, and `useStore` everywhere in component. In test, it looks OK.

When I publish package, and use it, all `inject` returns `undefined`. the custom component can not run normally.

It can not use like that or what do I need to do?

**Comment by wulinsheng123** [comment]
In development the currentInstance got object , when in production that's value got null

**Comment by wulinsheng123** [comment]
@jeremyjone Do you know how to reproduction?

**Comment by jeremyjone** [comment]
I solved the problem. Just fix the version number to `v2.6.14`. Because npm is automatically installed in the latest version, which is `v2.7.x`. They are in conflict.
